hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/EthereumCallDataHydration.java

```diff
@@ -74,15 +74,23 @@ public HydratedEthTxData tryToHydrate(
             if (callDataFile.deleted()) {
                 return failureFrom(FILE_DELETED);
             }
+
+            // Bytes.fromHex() doesn't appreciate a leading '0x' but we supported it in mono-service
```

Comment:
Mmm is this duplicated from [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/HevmTransactionFactory.java#L356)? Would prefer extracting a reusable method. 😇

Reply:
Created a tech debt story: https://github.com/hashgraph/hedera-services/issues/13226